frontend/packages/knative-plugin/src/utils/get-knative-resources.ts

```diff
@@ -24,6 +31,20 @@ const getKsResource = (dc: K8sResourceKind, res: K8sResourceKind): K8sResourceKi
   }
   return ksResource;
 };
+const mapStateToProps = (state: RootState): any => {
```

Comment:
If this is mapping some State to some Props, shouldn't there be an actual Props interface for this? 
That should give you a type instead of `any`